src/logging.cpp

```diff
@@ -10,7 +10,12 @@
 namespace Generators {
 
 LogItems g_log;
-static std::ostream* gp_stream{&std::cerr};
+
+static std::ostream*& GlobalLogStreamPtr() {
+  static std::ostream* stream = &std::cerr;
```

Comment:
avoid static initialization order race by making this a function. on Windows, I encountered the case where gp_stream was nullptr.

Reply:
The root problem is that our onnxruntime header was not supposed to have any dependencies and is now calling genai code, I forget what calls Ort:: functions first but if there's a way to clean that up, we should do that.